ci/docker_sources/hil_container/setupdb.sh

```diff
@@ -0,0 +1,27 @@
+hil-admin run-dev-server -p 7000 &
```

Comment:
kind of confused, why are you running it here again when you already run gunicorn on port 7000 in `runhi.sh`

Reply:
This part is just to turn on hil so the database could get setup.  I've tried to keep it close as possible to the old hil container setup too

Reply:
wouldn't this complain that the something is already running on port 7000?

Reply:
well this bit gets run before the gunicorn server, so it should be the gunicorn one that's complaining. Since we throw out the output though I think we just don't see it

Reply:
okay. I am fine with either running the flasks' built-in dev server (using the hil-admin command) or the gunicorn server. But we shouldn't run both because it leads to confusion and only one will fail anyway.

Reply:
I think I address this in the thread above, but these commands in setupdb get killed somewhere along the way before gunicorn & the second serve-networks call

Reply:
Yeah probably has to do with why runhil.sh has the `sleep infinity` in it.  I think when `setupdb.sh` exits even though there's the `&` it kills the servers.